### ISSUE TITLE
Separate components and pages

### ISSUE THREAD

**Issue by inezabonte** [EmanuelGF/template-react-ui#21]
In some projects I have seen a different approach being used towards separating components and pages
 - Usually a pages or views folder is used to store the actual pages on your website.
 - Components page is used to create components such as Navigation bars or footers which are then imported into a file in the pages folder.
 - Just a thought I had. You could consider it.🙂

**Comment by EmanuelGF** [comment]
You are correct. Thanks for pointing that out. 👍 Having the pages folder with the pages component will organize the project much better. 
Do you wanna have a go at doing this?

**Comment by inezabonte** [comment]
Sure I can take this 🙂

**Comment by EmanuelGF** [comment]
Thank you 🐱‍🏍
At the end don't forget to change the README folder structure, again... 😁
